From f1a267102eb56c52d7d79ec980bf7debe54dbc96 Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Wed, 14 Apr 2021 12:17:14 -0500
Subject: [PATCH 082/196] Update QAPP_* constants (#149)

This fixes a bug where the Qt wallet would use the old Bitcoin ABC datadir, causing all sorts of problems
---
 src/qt/guiconstants.h | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/qt/guiconstants.h b/src/qt/guiconstants.h
index 565f6c7b5..596f89282 100644
--- a/src/qt/guiconstants.h
+++ b/src/qt/guiconstants.h
@@ -42,11 +42,11 @@ static const int TOOLTIP_WRAP_THRESHOLD = 80;
 /* Number of frames in spinner animation */
 #define SPINNER_FRAMES 36
 
-#define QAPP_ORG_NAME "BitcoinABC"
-#define QAPP_ORG_DOMAIN "bitcoinabc.org"
-#define QAPP_APP_NAME_DEFAULT "BitcoinABC-Qt"
-#define QAPP_APP_NAME_TESTNET "BitcoinABC-Qt-testnet"
-#define QAPP_APP_NAME_REGTEST "BitcoinABC-Qt-regtest"
+#define QAPP_ORG_NAME "LogosFoundation"
+#define QAPP_ORG_DOMAIN "logos.cash"
+#define QAPP_APP_NAME_DEFAULT "Lotus-Qt"
+#define QAPP_APP_NAME_TESTNET "Lotus-Qt-testnet"
+#define QAPP_APP_NAME_REGTEST "Lotus-Qt-regtest"
 
 /* One gigabyte (GB) in bytes */
 static constexpr uint64_t GB_BYTES{1'000'000'000};
-- 
2.39.5 (Apple Git-154)

